boards/xtensa/esp32_ethernet_kit/board_init.c

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022 Grant Ramsay <grant.ramsay@hotmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/gpio.h>
+
+#define IP101GRI_RESET_N_PIN	5
```

Comment:
I _think_ this is correct. The phy_mii driver does not have dts bindings for a reset pin, as it's not part of the standard.
AFAIK this is a workaround for the wrover module using GPIO0 as a bootstrapping pin:

> To prevent the power-on state of the GPIO0 from being affected by the clock output on the PHY side, the RESET_N signal to PHY defaults to low, turning the clock output off. After power-on you can control RESET_N with GPIO5 to turn the clock output on